raw-cms-app/src/modules/core/components/left-menu/left-menu.js

```diff
@@ -46,6 +46,7 @@ const _LeftMenu = async (resolve, reject) => {
         this.$router.push({ name: item.route });
       },
       isActive: function(item) {
+        vuexStore.dispatch('core/updateTopBarTitle', this.$route.name);
```

Comment:
This shouldn't be here, because we don't use route names as titles for top bar: in fact, we always take them from vue-i18n to support translations. For example, for the collection list view, the title was: `$t('core.collections.title')`. You should remove this line and update the top bar title in each view where you have deleted the title from HTML template (as you did in "raw-cms-app/src/modules/core/views/user-details-view/user-details-view.js")